leader-common/src/main/kotlin/org/apache/amaterasu/leader/common/configuration/ConfigManager.kt

```diff
@@ -18,6 +18,10 @@ package org.apache.amaterasu.leader.common.configuration
 
 import com.uchuhimo.konf.Config
 import com.uchuhimo.konf.source.yaml.toYaml
+import org.apache.amaterasu.leader.common.configuration.dataset.DataSetsSpec
```

Comment:
This should be in its own class (maybe DatasetsManager?) and not a part of the ConfigurationManager. In addition, this should be under the frameworks/jvm-common or frameworks/java-common however we choose to name it.